rfcs/0010-additional-target-hooks.md

```diff
@@ -0,0 +1,155 @@
+Feature Name: additional-target-hooks
+Start Date: 2021-07-14
+RFC PR: apache/tvm-rfcs#10
+GitHub Issue: apache/tvm#8589
+
+# Summary
+[summary]: #summary
+
+In order to enable flexibility in how individual targets are lowered and built within TVM, this RFC proposes supporting additional hooks on the `Target` and that the target becomes the central place for such hooks, for example:
```

Comment:
could you clarify what you mean by "central place for such hooks?" Specifically, qualify what is needed of a hook that makes it a candidate for adding to this target hooks map. Since we are going to a world where everything is a compiler pass, it would be great to briefly discuss the difference between this hook (used as a plug-in to LowerTE scheduler) and other passes a Target may want to add to the compilation flow.

Also, i think our discussion resolved towards that we don't want to encourage the hook to be the central way people make target-specific modifications to TIR. However, this text reads that way. Can you update it?